src/config/typeorm.config.ts

```diff
@@ -16,13 +16,27 @@
 
 import { TypeOrmModuleOptions } from '@nestjs/typeorm';
 
+export default () => ({
+
+  database: {
+    type: 'postgres',
+    host: 'localhost',
+    port:  5432,
+    username: 'postgres',
+    password: 'postgres',
+    database: 'timesheets',
+    entities: [__dirname + '/../**/*.entity.{js,ts}'],
+    synchronize: true,
+  }
+});
+
 /**
  * Config for database connetion.
  */
 export const typeOrmConfig: TypeOrmModuleOptions = {
   type: 'postgres',
```

Comment:
should these be referring the "config" values?

Reply:
No. I was following the documentation to implement through this file but it does not seem to work. So I implemented it another way, and have left this as I want to take another look in the future.

Reply:
ok - thanks